### PR TITLE
migrating 2015* posts to md

### PR DESCRIPTION
**Pull request by tpd-opitz**
I used this sed script to convert most html syntax to their markdown replacement

     sed -e 's/<\/\?\(i|em\(phase\)?\)>/*/gI' \
         -e 's/<\/\?\(b|strong\)>/**/gI'\
         -e 's/<p\( [^>]*\)\?>/\n\n/gI'\
         -e 's/<\/p>//gI'\
         -e 's/<a[^>]* href="\([^"]\+\)".*>\([^>]\+\)\(<\/[^a][^>]*>\)*<\/a>/[\2](\1)/gI'

### CODE REVIEW

**Comment by tpd-opitz** [comment]
I used this sed script to convert most html syntax to their markdown replacement

sed -e 's/<\/\?\(i|em\(phase\)?\)>/*/g' 's/<\/\?\(b|strong\)>/**/g' -e 's/<p\( [^>]*\)\?>/\n\n/g' -e 's/<\/p>//g'  -e 's/<a[^>]* href="\([^"]\+\)".*>\([^>]\+\)\(<\/[^a][^>]*>\)*<\/a>/[\2](\1)/g'

**Comment by jdvr** [comment]
I have been using a nodejs script with this this module (https://github.com/domchristie/to-markdown)

The script is pretty simple:
```js
#!/usr/bin/nodejs

var fs = require('fs');
var path = require('path');
var toMarkdown = require('to-markdown');


function readFile(filename, onFileContent, onError) {
  fs.readFile(filename, 'utf-8', function(err, content) {
    if (err) {
      onError(err);
      return;
    }
    onFileContent(filename, content);
  });
}



readFile(process.argv[2], function(filename, content) {
  //toMarkdown(stringOfHTML, { converters: [converter1, converter2, …] });
  var newFileName = path.basename(filename, '.html') + ".md";
  fs.writeFile(newFileName, toMarkdown(content), function(err) {
    if(err) {
        return console.log(err);
    }

    console.log("The file was saved!");
    fs.unlink(filename)
  });

}, function(err) {
  throw err;
});
```
It receive a html file, create a new md file with new content and remove the old. The only problem is with the yml at the start of the file, after run the script I have to ad new lines manually.

**Comment by jdvr** [comment]
btw the file _posts/2015-07-20-self-organizing-team-2-footballs.html has already been discussing on this pr #47

**Comment by tpd-opitz** [comment]
> it receive a html file, create a new md file with new content and remove the old.

I rather like the file rename as a separate commit so that git knows about it. 

> the file _posts/2015-07-20-self-organizing-team-2-footballs.html has already been discussing on this pr #47

Thanks for pointing there.

I think loosing  the underlining and the centering is the kind of *acceptable breaking changes*  Raimo (@rradczewski ) was talking about in his request #36. 

bye
Thomas

**Comment by jdvr** [comment]
If @tpd-opitz  is migrating 2015* files I think #47 should be close. What do you think @rradczewski ?

There is a lot of post, I can continue with the 2011* as I have done on #56

File: _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+author: JosephYao
+authorname: Joseph Yao
+title: APAC Day of Coderetreat on Jun 13
+preview: 'As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture.'
+---
+
```
Comment (rradczewski) [inline _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md]: Looks like the script adds a lot of new lines where it shouldn't

File: _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture. After I talked to other GDCR organisers in China and Asia, the idea was accepted quite well. Then, the date was set at Jun 13 (Saturday).
```
Comment (rradczewski) [inline _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md]: I think we should change this to craftsman *[ship]* culture.

File: _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+We now have 14 cities registered for this event and I created a map  [here](https://www.google.com/maps/d/viewer?mid=zFRRheM6-9AQ.k8wlhsC1wCNY&amp;usp=sharing).  
+
+
+![embedded map](https://www.google.com/maps/d/u/0/embed?mid=1lsAFrsPkiHnp0vUpqW_50NJQiuI)
```
Comment (rradczewski) [inline _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md]: We can keep the `iframe` here so the map appears in the article, right now (see [here](https://deploy-preview-59--hairdresser-umbrellas-72171.netlify.com/blog/2015/06/12/apac-day-of-coderetreat-on-jun-13/)) the article doesn't really look too nice

File: _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+
+P.S. The intention is to have a light way regional coderetreat event in APAC. So, there will be less material and activities required comparing to GDCR. E.g. there is no plan to have sponsor for the whole event, no printout, no slides, etc. Every location will organise the event on their own and we’re on the same day. And, this event is an addition to GDCR, not a replacement.
+
+
+P.P.S. Here is an [article link](http://blog.csdn.net/uxyheaven/article/details/46488069) (in Chinese) and a picture of article posted in a company's internal magazine.
+
+
+![APAC day of coderetreat]({site.url}/images/APACdayofCoderetreat.jpg)
```
Comment (rradczewski) [inline _posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md]: I don't think `site.url` is needed here. Jekylls [`link`](http://jekyllrb.com/docs/templates/#link) will make sure resources are valid when jekyll compiles the page, so I'd rather use that instead.

**Comment by rradczewski** [comment]
In a general matter, I'd prefer a squashed commit history for these kind of tasks in any case @tpd-opitz.

Regarding the conversion: Please don't do this in bulk but rather see if the posts need some more care. Markdown is supposed to ease maintenance and contributions, not dogmatically striping embeds and visual key styles from the posts altogether.

Thanks again for your work fellas 🎉

**Pull request closed without merging**